modules/rest-api/src/main/java/org/wso2/carbon/mdm/api/MobileDevice.java

```diff
@@ -94,14 +95,15 @@
 	 * Fetch Android device details of a given user.
 	 *
 	 * @param user   User Name
+	 * @param tenantDomain tenant domain
 	 * @return Device
 	 * @throws org.wso2.carbon.mdm.api.common.MDMAPIException
 	 *
 	 */
 	@GET
-	@Path("{user}")
-	public List<org.wso2.carbon.device.mgt.common.Device> getDevice(@PathParam("user") String user)
-			throws MDMAPIException {
+	@Path("{user}/{tenantDomain}")
```

Comment:
Is it required to have tenant domain at the path it self? cannot we retrieve those info based on the user?